src/gavel2html.coffee

```diff
@@ -60,6 +63,8 @@ class Gavel2Html
       @usedErrors = converter.usedErrors
       return result
     catch e
+      console.log('\n\nERROR CAUGHT:', e)
+      console.log('\n\n')
```

Comment:
do we want gavel2html to print to e.g. browser console?

Reply:
I would add at least `console.error` to know more details about the exception. Previously there's been no way to see the error since it just output "Internal validation error" in the html.